Pod/Classes/SEGAdobeIntegration.h

```diff
@@ -40,7 +40,7 @@
 @property (nonatomic, strong, nullable) ADBMediaObject *mediaObject;
 @property (nonatomic, strong, nullable) id<SEGADBMediaObjectFactory> ADBMediaObjectFactory;
 
-- (instancetype _Nonnull)initWithSettings:(NSDictionary *_Nonnull)settings;
-- (instancetype _Nonnull)initWithSettings:(NSDictionary *_Nonnull)settings andADBMobile:(id _Nullable)ADBMobile andADBMediaHeartbeatFactory:(id<SEGADBMediaHeartbeatFactory> _Nullable)ADBMediaHeartbeatFactory andADBMediaHeartbeatConfig:(ADBMediaHeartbeatConfig *_Nullable)config andADBMediaObjectFactory:(id<SEGADBMediaObjectFactory> _Nullable)ADBMediaObjectFactory;
+//- (instancetype _Nonnull)initWithSettings:(NSDictionary *_Nonnull)settings;
+- (instancetype _Nonnull)initWithSettings:(NSDictionary *_Nonnull)settings andADBMobile:(id _Nullable)ADBMobileClass andADBMediaHeartbeat:(ADBMediaHeartbeat *_Nullable)ADBMediaHeartbeat andADBMediaHeartbeatFactory:(id<SEGADBMediaHeartbeatFactory> _Nullable)ADBMediaHeartbeatFactory andADBMediaHeartbeatConfig:(ADBMediaHeartbeatConfig *_Nullable)config andADBMediaObject:(ADBMediaObject *_Nullable)mediaObject andADBMediaObjectFactory:(id<SEGADBMediaObjectFactory> _Nullable)ADBMediaObjectFactory;
```

Comment:
Why is this passing both `ADBMediaHeartbeat` and `ADBMediaHeartbeatFactory`? I understand you need one for tests and for the real implementation, but the factory should handle both cases.